service/src/main/java/uk/gov/hmcts/reform/fpl/model/CaseData.java

```diff
@@ -26,6 +28,7 @@
 @Data
 @Builder(toBuilder = true)
 @AllArgsConstructor
+@IsIncludedInSwet(groups = UploadDocumentsGroup.class)
```

Comment:
Not sure if the name of the annotation is correct, case data is not included in SWET. Do you think we could make the naming explicit? 
@HasDocumentsIncluded in Swet or something like that?

Reply:
Yep ok makes sense. Renamed to HasDocumentsIncludedInSwet